packages/rspeedy/plugin-react-alias/src/index.ts

```diff
@@ -40,6 +42,22 @@ export function pluginReactAlias(options: Options): RsbuildPlugin {
           paths: [rootPath ?? api.context.rootPath],
         }),
       )
+
+      const pkgPath = path.join(reactLynxDir, 'package.json')
+      const pkgContent = readFileSync(pkgPath, 'utf-8')
+
+      let version: string | undefined
+      try {
+        const pkg = JSON.parse(pkgContent) as { version?: string }
```

Comment:
It would be simplier to use `require()` instead of `readFileSync` + `JSON.parse()`.

BTW, we already have the absolute path above using `require.resolve('@lynx-js/react/package.json)`. We may re-use that.